src/guides/v2.4/ui_comp_guide/components/ui-sortby.md

```diff
@@ -0,0 +1,71 @@
+---
+group: ui-components-guide
+title: Sortby component
+contributor_name: Shankar Konar
+contributor_link: https://github.com/konarshankar07
+---
+
+The Sortby component allows you to sort the column in ascending or descending order.
+
+## Configuration options
+
+|      Option                |   Description                                                                 |      Type      |  Default Value   |
+|----------------------------|-------------------------------------------------------------------------------|----------------| -----------------|
+| `template`                 | Path to the component .html template.                                         | String         | `ui/grid/sortBy` |
+| `options`                  | List of options available for sorting.                                        | Array          | `-`              |
```

Comment:
If this is not an actual default, you don't need the inline code formatting for the `-`.

```suggestion
| `options`                  | List of options available for sorting.                                        | Array          | `-`              |
```